.github/actions/maven-update-pom-version/action.yml

```diff
@@ -16,7 +16,7 @@ inputs:
 outputs:
   next-prerelease:
     description: "Next prerelease"
-    value: ${{ steps.next-prerelease-resolver.outputs.next-prerelease }}
+    value: ${{ steps.resolve-version.outputs.version }}
```

Comment:
I think this is a left over from a previous change. There is no step with id `resolve-version` in this action, so all this block can be removed.

Reply:
Removed the whole block, thank you for spotting this!